lib/tasks/curator.rake

```diff
@@ -0,0 +1,21 @@
+namespace :curator do
+  desc <<-DESC.strip_heredoc
+    Sets the curator for a given scope, curated highlights are available by querying the set
+    "curated:openstax".
+
+    eg: rake curator:set[scope_uuid,curator_uuid]
+  DESC
+  task :set, [:scope_id, :curator_id] => :environment do |task, args|
+    CuratorScope.where(scope_id: args[:scope_id]).destroy_all
+    CuratorScope.create(args.to_hash)
```

Comment:
who runs this?  should we check the args before they're passed into destroy and create?  (e.g., could someone mess this up badly?)